src/System.Private.CoreLib/shared/System/Globalization/CalendarData.cs

```diff
@@ -51,7 +52,28 @@ internal partial class CalendarData
         internal static readonly CalendarData Invariant = CreateInvariant();
 
         // Private constructor
-        private CalendarData() { }
+        private CalendarData()
+        {
+            // TODO-NULLABLE: this class requires refactoring for proper annotations
+            //                currently from user of this class all fields are non-nullable. To avoid potential breaking changes lots of workaround have
+            //                been used to suppress errors
+            sNativeName = null!;
```

Comment:
You should do this on the field declaration. That way the compiler will not emit this into IL latter.